Basic/Invaders/Assets/Scripts/EnemyBullet.cs

```diff
@@ -74,15 +74,15 @@ private void OnTriggerEnter2D(Collider2D collider)
         var hitShield = collider.gameObject.GetComponent<Shield>();
         if (hitShield != null)
         {
-            SpawnExplosionVFXClientRpc(transform.position, Quaternion.identity);
+            ClientSpawnExplosionVFXRpc(transform.position, Quaternion.identity);
 
             Destroy(hitShield.gameObject);
             NetworkObject.Despawn();
         }
     }
 
-    [ClientRpc]
-    void SpawnExplosionVFXClientRpc(Vector3 spawnPosition, Quaternion spawnRotation)
+    [Rpc(SendTo.ClientsAndHost)]
+    void ClientSpawnExplosionVFXRpc(Vector3 spawnPosition, Quaternion spawnRotation)
```

Comment:
In order to maintain consistency with NGO's naming (and with the gaming industry's best practice), methods in NetworkBehaviours that are executed on Client or Server should start with "OnClient" or "OnServer" for clarity.

@bastienunity Since you're going through renaming RPCs, could you rename according to that logic?

Reply:
Hey Paolo. This suggestion seems to contradict the alignment we had with Noel in [this discussion](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/pull/883#discussion_r1531301114), at least for Rpcs.

As for the other methods, if it's not listed on our coding standard reference doc, we don't necessarily need to follow any naming convention. If we want to introduce that as a convention for our samples moving forward, we should establish that convention first, since that's not present in any sample so far.

Reply:
The alignment we had with Noel in that discussion was about keeping a prefix, which is maintained in both cases. How does my previous comment contradictit?

The additional "On" prefix is what is present in NGO's and Small Scale template's codebases. As a user, I'd expect netcode code to stay consistent with the naming convention of the netcode framework I'm using.

Reply:
The alignment in that conversation was to name Rpcs invoked on clients or server as just Client<MethodName>Rpc, and Server<MethodName>Rpc. This line is a similar scenario, no?

We haven't established yet to add the prefix `On` on methods invoked on clients, server, or authority yet. We're starting those conversations offline, so let's align there!

Reply:
@RikuTheFuffs Let's proceed with the current Rpc naming and come back to refactors, to try and get this PR through.